### ISSUE TITLE
✨ Feat: Improve UI for the SAML IdP selection list

### ISSUE THREAD

**Issue by Millefeuille42** [tracim/tracim#6242]
### Feature description

This concerns the list and the "back" button as seen on the screenshot.
This is about code brought by the yet unmerged https://github.com/tracim/tracim/pull/6232

Current UI is poorly done with already present elements and needs a makeover.

![image](https://github.com/tracim/tracim/assets/42064302/ce7694ef-2e8f-40b1-baa0-3b86231df78e)
![image](https://github.com/tracim/tracim/assets/42064302/875dd3b4-2522-4dc7-a73d-2cc4106d40fb)

### How it works

Concerned page and elements are featured in `.../frontend/src/component/Login/SignIn.jsx`and `.../frontend/src/css/Login.styl`.

### Related PR

Originates from and must target https://github.com/tracim/tracim/pull/6232

**Comment by lebouquetin** [comment]
@Millefeuille42 I see the classical login link. Does it work even while using SAML? Is it related to authentication type list defined in `.ini` config file ?

`auth_types = internal, remote, ldap, saml`?

**Comment by Millefeuille42** [comment]
Yes it only appears when SAML is set as a auth_type in the `.ini` file.
If you are coming for a different branch, don't forget to rebuild the `frontend` module too.

Additionally, reset your browser cache with `ctrl+shift+r`

**Comment by lebouquetin** [comment]
To be more explicit, the behavior I expect is the following one:

- if `standard` in `auth_types` -> no list of IdP, show standard login form
- if `saml` in `auth_types` and `standard` in `auth_types` AND `saml` BEFORE `standard` -> show list of IdP + link to normal form
- if `saml` in `auth_types` and `standard` in `auth_types` AND `saml` AFTER `standard` -> show normal form + link to list of IdP
- if  `saml` in `auth_types` and `standard` NOT in `auth_types` -> show only list of IdP

This is a very global behavior which works in all cases. The only acceptable shortcut is to mix 2nd and 3rd cases

**Comment by lebouquetin** [comment]
Here is the expected result:

![login-tracim-saml](https://github.com/tracim/tracim/assets/907161/0c253cb3-b929-4f10-91f5-c4e54c673105)

- the second button (classical login) should appear only if standard auth is configured
- the icons must be left aligned (image should have a max width so that it wont overwrite the label)
- the labels must be left aligned
- the clickable zone is both icon + label
- the title associated to the link should show the label + the targeted url (eg `Université du littoral - https://idp.univ-littoral.fr`)

**Comment by lebouquetin** [comment]
colors of the screenshot are not the good one (keep the standard colors)

**Comment by Millefeuille42** [comment]
Ok I see. Currently Behavior 1 and 4 is implemented.
Just need to implement behavior 2 and 3

**Comment by Skylsmoi** [comment]
fixes are in https://github.com/tracim/tracim/pull/6232
